### PR TITLE
select2 4.0.2 -> 4.0.3

### PR DESCRIPTION
**Pull request by kimyu92**
* Bump select2 from `4.0.2` to `4.0.3`
* Update select2 css to match official select2 upstream
* Update select2 bootstrap css `0.1.0-beta.4` -> `0.1.0-beta.7`
* Add `el` and `km` locales
* Connected to #161

### CODE REVIEW

**Comment by kimyu92** [comment]
cc @argerim @lunks

**Comment by kevin-jj** [comment]
input-lg in select2 bootstrap does not works

**Comment by kimyu92** [comment]
@kevin-jj Based on the [documentation](https://select2.github.io/select2-bootstrap-theme/4.0.0.html)

Have you tried to include `//= require select2-full` rather than `//= require select2`

**Comment by kevin-jj** [comment]
It works. Thank you so much.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+**/.DS_Store
```
Comment (simi) [inline .gitignore]: :scissors: please

**Comment by kimyu92** [comment]
@simi anything else?

**Comment by simi** [comment]
giphy 11

**Pull request merged**